pseudo_dojo/refdata/hb/__init__.py

```diff
@@ -0,0 +1 @@
+from .database import hb_database, hb_compute
```

Comment:
Please remove hb files in the next PR, we are gonna create another repo to host the data